### PR TITLE
Container added for RO-OHT and change of color based on TDS Value

### PR DESCRIPTION
**Pull request by perireddyvaka**
Color changing functionality added to the sump water based on the TDS Value

### CODE REVIEW

**Review by perireddyvaka — COMMENTED** [review]
created the RoOHT as like KRBOHT

**Comment by likhithkanigolla** [comment]
@perireddyvaka I have merged the Conficts, Please Checkout with GitHub CLI of this PR and let me know if anything breaking from your end.

File: src/components/navigation/Navigation.jsx
Comment (likhithkanigolla) [inline src/components/navigation/Navigation.jsx:1]: This is the fix for the MUI

**Review by likhithkanigolla — APPROVED** [review]
Approving Changes

**Pull request merged**